org/commits.test.js

```diff
@@ -1,91 +1,77 @@
+  ],
 
-    lineLength();
-    expect(global.fail).not.toBeCalled();
-  });
+  commitVerb: [
+    {actual: mockDanger([]), expected: []},
+    {actual: mockDanger([mockCommit('Add new feature')]), expected: []},
+    {actual: mockDanger([mockCommit('Provoke new feature')]), expected: []},
+    {actual: mockDanger([mockCommit('add new feature')]), expected: [noVerb]},
+    {actual: mockDanger([mockCommit('feat: 💩 I like emoji!!!')]), expected: [noVerb]},
+  ],
 
-  it('should ignore merge commits', () => {
-    setCommits([
-      //eslint-disable-next-line max-len
-      buildCommit('Merge pull request #42 from wearereasonablepeople/OurOrgNameIsAlreadyPrettyLong/so-merge-commits-can-be-huge')
-    ]);
+  commitMessageLength: [
+    {actual: mockDanger([]), expected: []},
+    {actual: mockDanger([mockCommit('small message')]), expected: []},
+    {actual: mockDanger([mockCommit(repeat('lo', 100).join())]), expected: [
+      'Commit [66d891](./commits/66d8911a) has lines with over 70 characters.'
+    ]}
+  ],
 
-    lineLength();
-    expect(global.fail).not.toBeCalled();
-  });
-});
-
-describe('format', () => {
-  it('should warn to add prefix', () => {
-    setCommits([
-      buildCommit('feat: a prefix'),
-      buildCommit('no prefix', '1234567890'),
-    ]);
+  commitMessageAscii: [
+    {actual: mockDanger([]), expected: []},
+    {actual: mockDanger([mockCommit('foo bar')]), expected: []},
+    {actual: mockDanger([mockCommit('!@#$%^&*(){}[]')]), expected: []},
+    {actual: mockDanger([mockCommit('feat: 💩 I like emoji!!!')]), expected: [
+      'Message header for commit [66d891](./commits/66d8911a) must contain ASCII characters only.'
+    ]},
+  ],
 
-    typePrefix();
-    expect(global.warn).toBeCalledWith(expect.stringContaining('12345'));
-  });
-  it('should be okay when all have a prefix', () => {
-    setCommits([
-      buildCommit('feat: a prefix'),
-      buildCommit('Feat: also case insensitive'),
-      buildCommit('ci: another one'),
-    ]);
-
-    typePrefix();
-    expect(global.warn).not.toBeCalled();
-  });
-  it('should ignore Merge commits', () => {
-    setCommits([
-      buildCommit('Merge pull request #42 from wearereasonablepeople/anyone/do-stuff'),
-      buildCommit('feat: a prefix'),
-    ]);
+  commitMessagePunctuation: [
+    {actual: mockDanger([]), expected: []},
+    {actual: mockDanger([mockCommit('foo bar')]), expected: []},
+    {actual: mockDanger([mockCommit('feat: 💩 I like emoji!!!')]), expected: [
+      'Message header for commit [66d891](./commits/66d8911a) must ' +
+      'end in an alphanumerical character.'
+    ]},
+  ],
+};
 
-    typePrefix();
-    expect(global.fail).not.toBeCalled();
-    expect(global.warn).not.toBeCalled();
+Object.keys(testAssertions).forEach(testcase => {
+  describe(testcase, () => {
+    testAssertions[testcase].forEach(({actual, expected}, i) => {
+      it(`passes test ${i + 1}`, () => {
+        expect(tests[testcase].test(actual)).toEqual(expected);
```

Comment:
Nice structure, want to refactor everything when I have time, though I'd say `actual` is a `fixture` rather than the actual output.